frontend/cypress/integration/demo/demo.spec.ts

```diff
@@ -40,6 +40,7 @@ describe('demo', () => {
                     res.body.flags = {
                         ...res.body.flags,
                         demo: true,
+                        flagOverviewRedesign: true,
```

Comment:
What's the implications of this?

Reply:
This makes the demo e2e tests, that we only run manually and rarely, use the new flag overview page redesign.